src/molecules/lists/IconList/index.js

```diff
@@ -35,6 +35,7 @@ function IconList(props) {
               <Icon
                 className={styles.icon}
                 icon={item.icon}
+                lazy={typeof item.lazy === 'undefined' ? true : item.lazy}
```

Comment:
What's your thought on just passing this as a defaultProp?

Reply:
@danielnovograd I don't think you can pass in a `defaultProp` for a specific value in an object, the object here being the `data` prop.